### DISCUSSION TITLE
23.11.09 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#59] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 발표 준비
    - 리뷰 확인 및 코드 수정 (진행 중)
- 내일은 어떤 일을 하나요?
    - 발표 준비

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
- 내일은 어떤 일을 하나요?
    - 노래 : 스포티파이 API와 연동해서 데이터 가져오기

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * PR 코드 리뷰
* 내일은 어떤 일을 하나요?
  * 발표준비

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 디버깅 완료 후 merge
  - 비동기 테스트 다른 방법 시도
  - 리뷰하기
- 내일은 어떤 일을 하나요? 
  - 발표 준비
